tst/unit/test_sparse_pack.cpp

```diff
@@ -77,13 +77,76 @@ struct v5 : public parthenon::variable_names::base_t<false> {
   static std::string name() { return "v5"; }
 };
 
+struct v7 : public parthenon::variable_names::base_t<false, 3, 3> {
```

Comment:
I just thought of this---maybe we should do `ANYDIM, 3` here, instead of `3, 3` to show that the slowest moving index can be arbitrary size